TurkishFilter/sections/specific.txt

```diff
@@ -425,9 +425,8 @@ koinmedya.com##a[href^="https://beincrypto.com/"]
 pembetv18.com,kriptoparafiyatlari.com,koinmedya.com##a[href^="https://www.binance.com/"]
 koinmedya.com###custom_html-52
 ||dizibox.*/r/
-||i.ibb.co/2Pt8tqJ/turna.png
 ||i.ibb.co/0nCpbCv/otelz.jpg
-||i.ibb.co/bXc58hB/turna.png
+||i.ibb.co/*/turna*$image
```

Comment:
second `*` is redundant

Reply:
There are two different file name from each other. Someone is turnaa.png and the other turna.png. They both belong to the same advertisement, but they have different file names. With the odd line is both unblocked. Thus, two ads are blocked by single line.

Reply:
https://i.ibb.co/bXc58hB/turna.png
https://i.ibb.co/tJ9cJVx/turnaa.png

Reply:
`||i.ibb.co/*/turna` covers both cases.
Or maybe
```
||i.ibb.co/*/turna.png
||i.ibb.co/*/turnaa.png
```

Reply:
Thanks for the explanation. I updated PR.